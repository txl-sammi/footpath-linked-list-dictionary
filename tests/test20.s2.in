10.4
19.45
21.2
27.2
29.4
31.3
31.7
37.8
50.7
58.3
61.7
62.4
65.79
79.81
101.4
118.6
120.5
158.2
174.5
206.91
