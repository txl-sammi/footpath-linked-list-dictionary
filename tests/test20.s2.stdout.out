10.4 --> 10.5
19.45 --> 19.5
21.2 --> 21.3
27.2 --> 27.3
29.4 --> 29.5
31.3 --> 31.4
31.7 --> 31.8
37.8 --> 37.9
50.7 --> 50.8
58.3 --> 58.2
61.7 --> 61.8
62.4 --> 62.5
65.79 --> 65.8
79.81 --> 79.8
101.4 --> 101.3
118.6 --> 118.5
120.5 --> 120.4
158.2 --> 158.1
174.5 --> 174.4
206.91 --> 206.9
